### DISCUSSION TITLE
Erro nos capitulo de Modulos e dyynamic

### DISCUSSION THREAD

**Discussion by ralphtacconi** [chgasparoto/curso-aws-com-terraform#98] [Q&A]
Olá CLeber, estou tendo erros na seção de módulos: Quando plan no projeto ele me da o erro:
with aws_s3_bucket.this,
│   on main.tf line 1, in resource "aws_s3_bucket" "this":
│    1: resource "aws_s3_bucket" "this" {
│
│ Can't configure a value for "website": its value will be decided automatically based on the result of applying this configuration.

Isso em vários lugares. Pode me ajudaR?

**Comment by chgasparoto** [comment]
@ralphtacconi qual a versão do **aws provider**? Eu vi que lançaram a versão 4 e está gerando vários erros. No curso eu utilizo a versão [3.23](https://github.com/chgasparoto/curso-aws-com-terraform/blob/master/02-terraform-intermediario/05-modules/main.tf#L7)

Aqui você pode ver a [issue](https://github.com/hashicorp/terraform-provider-aws/issues/23125) no repo do provider
